lib/rules/fluent-chaining.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * @fileoverview Enforce formatting for chained calls
+ */
+
+'use strict';
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = {
+   create: function(context) {
+      var sourceCode = context.getSourceCode(),
+          lines = sourceCode.getLines();
+
+      //--------------------------------------------------------------------------
+      // Helpers
+      //--------------------------------------------------------------------------
+
+      function lineIndent(lineNumber) {
+         var line = lines[lineNumber - 1];
+
+         if (line === undefined) {
+            throw new Error('Invalid lineNumber ' + lineNumber + ' supplied to lineIndent');
+         }
+
+         return line[0] === ' ' ? line.match(/[^ \t]/).index : 0;
+      }
+
+      function validateSingleLineMemberExpression(node) {
+         if (node.object.loc.start.line !== node.property.loc.start.line) {
+            context.report({
+               node: node.property,
+               message: 'Identifier {{ identifier }} should be on a new line',
+               data: {
+                  identifier: node.property.name
+               },
+               fix: function(fixer) {
+                  var firstLineIndent = lineIndent(node.object.loc.end.line),
+                      fixString = '\n' + ' '.repeat(firstLineIndent) + '.';
```

Comment:
if the code above in `lineIndent` works with tabs or spaces, this would also need to do so